.github/workflows/static-checks.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - name: packaging check
         run: |
           ./.github/scripts/setup_generate_license.sh
-          ${MVN} clean install -Prat -Pbundle-contrib-exts --fail-at-end \
```

Comment:
Why was this change needed?

Reply:
im not sure we should remove it, the packaging check was added to make sure that everything builds correctly, including contrib extensions (at least that is how i remember it)

Reply:
Keeping this profile in the command leads to misleading warning messages such as this one:
https://github.com/apache/druid/actions/runs/4261844965/jobs/7416643271#step:5:22535
```
Warning:  The requested profile "bundle-contrib-exts" could not be activated because it does not exist.
```
This is because the profile `bundle-contrib-exts` is present only in the distribution module and that module has been excluded in the command here.
https://github.com/apache/druid/blob/6bb5effa7bb1c7a31cff270818d035bdfb7a7883/.github/workflows/static-checks.yml#L56-L62

I have retained the `-Pbundle-contrib-exts` for the second command, where the distribution module has been targeted.

Reply:
ah, thanks I was confused, as long as it is still building them for the distribution :+1: